tests/test_context.py

```diff
@@ -94,11 +85,12 @@ def test_speed_of_sound(device):
 
 
 def test_distance_model(device):
+    """Test preset values distance_model"""
     with Context(device) as context:
         context.distance_model = DistanceModel.INVERSE_CLAMPED
         context.distance_model = DistanceModel.LINEAR_CLAMPED
         context.distance_model = DistanceModel.EXPONENT_CLAMPED
         context.distance_model = DistanceModel.INVERSE
         context.distance_model = DistanceModel.LINEAR
         context.distance_model = DistanceModel.EXPONENT
-        context.distance_model = DistanceModel.NONE
+        context.distance_model = DistanceModel.NONE
```

Comment:
Oh wait I fucked up this

Reply:
Also, @McSinyx: How to get those preset values for `distance_model`?

Reply:
`git pull origin master`

Reply:
I already cherry picked that enum implementation. I just leave this part here bc Idk how to do it 😆

Reply:
The proper way to do it is `git pull --rebase origin master`

Reply:
NO, i mean how to get those preset values for distance_model

Reply:
Oops sorry didn't read it through, they're in `palace.distance_models`.